runtime/src/lib.rs

```diff
@@ -158,6 +158,9 @@ pub fn native_version() -> NativeVersion {
 }
 
 pub struct BaseFilter;
+#[cfg(feature = "testing-node")]
+mod testing_runtime;
```

Comment:
no reason to really separate this to file when only this filter is overridden, either both should be extracted or neither